Project.toml

```
@@ -7,20 +7,24 @@ version = "0.1.3"
 Dictionaries = "85a47980-9c8c-11e8-2b9f-f7ca1fa99fb4"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 PEG = "12d937ae-5f68-53be-93c9-3a6f997a20a8"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Aqua = "0.6"
 Dictionaries = "0.3"
 DocStringExtensions = "0.9"
 LightXML = "0.9"
+LinearAlgebra = "1.11.0"
```

```suggestion
LinearAlgebra = "1.6"
```

since we want the library to work also on older julia versions (starting to 1.6) we want to set the lower bound to `1.6`, similar for `Statistics`

---

also I think you'll need `]add Random` and add a lower bound (also 1.6)